src/ophys_etl/filters/filters.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+from scipy.sparse import coo_matrix
+
+
+def filter_rois_by_longest_edge_length(coo_rois: List[coo_matrix],
+                                       longest_edge_threshold: int):
```

Comment:
Add the return type as well

`longest_edge_threshold: int) -> List[coo_matrix]:`

Reply:
Added return type hint